src/travertino/declaration.py

```diff
@@ -143,10 +160,53 @@ def __delete__(self, obj):
         else:
             obj.apply(self.name, self.initial)
 
+    @property
+    def _name_if_set(self):
+        return getattr(self, "name", "")
+
+    def validate(self, value):
+        try:
+            return self.choices.validate(value)
+        except ValueError:
+            raise ValueError(
+                f"Invalid value {value!r} for property {self._name_if_set}; "
```

Comment:
A minor cosmetic thing but this will print `... for property ;` if name isn't set. Given `_name_if_set` is private, it might be worth incorporating the separator space into the return value.

Reply:
I did that first, but then realized that `_name_if_set` is used in several places, with different spacing requirements. Do you think it's worth `" ".join`ing it in each error message?

Reply:
Scratch that, I figured out a way that works.